matrix/lambdas/api/core.py

```diff
@@ -1,23 +1,61 @@
-import json
-import os
 import requests
+import uuid
 
-import boto3
+from ...common.constants import MatrixFormat
+from ...common.constants import MatrixRequestStatus
+from ...common.lambda_handler import LambdaHandler
+from ...common.lambda_handler import LambdaName
 
-LAMBDA_CLIENT = boto3.client("lambda",
-                             region_name=os.environ['AWS_DEFAULT_REGION'])
 
+def post_matrix(body: dict):
+    has_ids = 'bundle_fqids' in body
+    has_url = 'bundle_fqids_url' in body
 
-def post_matrix(body):
-    LAMBDA_CLIENT.invoke(
-        FunctionName=os.environ['DRIVER_FN_NAME'],
-        InvocationType="Event",
-        Payload=json.dumps(body).encode(),
-    )
-    return "post_matrix", requests.codes.ok
+    format = body['format'] if 'format' in body else MatrixFormat.ZARR.value
 
+    # Validate input parameters
+    if not (has_ids or has_url):
```

Comment:
combo of not and or leads to high code complexity

Reply:
Will go with `not has_ids and not has_url`